doc/options.md

```diff
@@ -173,12 +173,11 @@ spec:
 ```
 
 ## `spec.expose`
-This section contains configuration for exposing Spinnaker. It is optional, if you omit it
-no load balancer will be created (or deleted if you remove it). 
+Optional. Controls how Spinnaker gets exposed. If you omit this section, no load balancer gets created. If this section is removed, the Load Balancer gets deleted. 
```

Comment:
If this section is removed, the Load Balancers are **not** deleted. This is because we want to support users to "opt out" of operator managed load balancers.

Reply:
Changed the sentence to 

> If this section is removed, the Load Balancer does not deleted.